hoomd/md/nlist.py

```diff
@@ -164,20 +190,75 @@ def gpu_local_nlist_arrays(self):
         Provides direct access to the neighbor list arrays on the gpu. All data
         is MPI rank-local.
 
-        The `hoomd.md.data.NeighborListLocalAccessGPU` object exposes the three
-        arrays to efficently iterate over the neighbor list.
+        The `hoomd.md.data.NeighborListLocalAccessGPU` object exposes the
+        internal data representation to efficently iterate over the neighbor
+        list.
 
         Note:
             The local arrays are read only.
 
         See Also:
-            `cpu_local_nlist_arrays` for an example of how the arrays are
-            accessed
+            `cpu_local_nlist_arrays`
 
         Examples::
 
-            with self.gpu_local_nlist_arrays as arrays:
```

Comment:
Thanks for such an in-depth example! Have you verified that this code works?

Reply:
Yes! I use this example in the tests for the local arrays!